ESPWebThingAdapter.h

```diff
@@ -242,9 +243,12 @@ class WebThingAdapter {
   }
 
   void serializeDevice(JsonObject& descr, ThingDevice* device) {
-    descr["name"] = device->name;
-    descr["href"] = "/things/" + device->id;
+    descr["id"] = device->id;
+    descr["title"] = device->title;
     descr["@context"] = "https://iot.mozilla.org/schemas";
+    descr["securityDefinitions"] = "{\"nosec_sc\": {\"scheme\": \"nosec\"}}";
```

Comment:
This is a JSON string, not a proper object (which will be a pain to create using ArduinoJSON). It might be worthwhile to put the nosec definition somewhere shared so you don't have to copy as much

Reply:
It was pretty easy. Updated.